### PR TITLE
formatting a lot of formatting, also added annotations

### PR DESCRIPTION
**Pull request by J3fftw1**
never * import
also, packages are lowercase

Formatted this a lot and i mean a lot

### CODE REVIEW

**Comment by J3fftw1** [comment]
please go over this slowly and dont just merge it i might made some errors

File: src/main/java/me/mooy1/infinityexpansion/Items.java [hunk 1]
```diff
@@ -1,429 +1,429 @@
...
-            "QUARRY",
-            Material.STONE_BRICKS,
-            "&7&lQuarry",
-            LoreBuilder.speed(1),
-            LoreBuilder.powerPerSecond(4_000),
-            ""
+        "QUARRY",
+        Material.STONE_BRICKS,
+        "&7&lQuarry",
+        LoreBuilder.speed(1),
```
Comment (Mooy1) [inline src/main/java/me/mooy1/infinityexpansion/Items.java:18]: i added another lore here, and at 27 36 45

Comment (J3fftw1) [reply src/main/java/me/mooy1/infinityexpansion/Items.java:18]: They arent there though
Idk what you mean in the current version they arent there and in my changes are the exact same as the old version just formatted

Comment (J3fftw1) [reply src/main/java/me/mooy1/infinityexpansion/Items.java:18]: Oh you updated it ah will fix when home

File: src/main/java/me/mooy1/infinityexpansion/machines/Quarry.java [hunk 2]
```diff
@@ -42,37 +39,38 @@ public Quarry(Tier tier) {
...
     }
 
 
     @Override
     public int getCapacity() {
-        return tier.getEnergyConsumption()*2;
+        return tier.getEnergyConsumption() * 2;
     }
 
     @Nonnull
```
Comment (Mooy1) [inline src/main/java/me/mooy1/infinityexpansion/machines/Quarry.java:76]: I also updated the names of the stained glass on 106, 111, 119

**Comment by J3fftw1** [comment]
I will merge your commits in this tonight 
And i will resolve conflicts

**Comment by J3fftw1** [comment]
fixed

File: src/main/java/me/mooy1/infinityexpansion/InfinityExpansion.java [hunk 3]
```diff
@@ -26,7 +26,7 @@ public void onEnable() {
         Config cfg = new Config(this);
 
         if (cfg.getBoolean("options.auto-update") && getDescription().getVersion().startsWith("DEV - ")) {
-            new GitHubBuildsUpdater(this, getFile(), "Mooy1/InfinityExpansion/master").start();
+            new GitHubBuildsUpdater(this, getFile(), "Mooy1/InfinitExpansion/master").start();
```
Comment (Mooy1) [inline src/main/java/me/mooy1/infinityexpansion/InfinityExpansion.java:29]:
```suggestion
            new GitHubBuildsUpdater(this, getFile(), "Mooy1/InfinityExpansion/master").start();
```

**Pull request merged**